components/invitation_modal/invitation_modal_guest_step/invitation_modal_guests_step.jsx

```diff
@@ -33,14 +34,12 @@ class InvitationModalGuestsStep extends React.PureComponent {
         onSubmit: PropTypes.func.isRequired,
         emailInvitationsEnabled: PropTypes.bool.isRequired,
         userLimit: PropTypes.string.isRequired,
-        currentUsers: PropTypes.number.isRequired,
         userIsAdmin: PropTypes.bool.isRequired,
         isCloud: PropTypes.bool.isRequired,
-        analytics: PropTypes.object.isRequired,
-        subscription: PropTypes.object.isRequired,
+        subscriptionStats: PropTypes.object,
         actions: PropTypes.shape({
-            getStandardAnalytics: PropTypes.func.isRequired,
-            getCloudSubscription: PropTypes.func.isRequired,
+            getSubscriptionStats: PropTypes.func.isRequired,
+            getCloudSubscription: PropTypes.func.isRequired, // required by the withGetCloudSubscription HOC
```

Comment:
Given that we don't need the subscription here anymore, maybe we don't need the HOC `withGetCloudSubscription` ?

Reply:
That's true.